services/QuillLMS/lib/tasks/queries.thor

```diff
@@ -1,7 +1,6 @@
 require_relative '../../config/environment'
 
```

Comment:
I think there is a mistaken merge here and this should take the files from `develop` (I think you should just check this file out from develop).

Reply:
I think @anathomical had added these changes to the branch-- Thomas, should I leave as is or pull from develop?